src/pages/sihyun/Login/Login.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [user, setUser] = useState({
+    email: '',
+    password: '',
+  });
+  const emailRegEx =
+    /^[A-Za-z0-9]([-_.]?[A-Za-z0-9])*@[A-Za-z0-9]([-_.]?[A-Za-z0-9])*\.[A-Za-z]{2,3}$/;
+
+  const updateForm = e => {
+    const updateUser = { ...user, [e.target.name]: e.target.value };
+    setUser(updateUser);
+  };
+
+  const handleLogin = () => {
+    fetch('http://10.58.52.161:8000/users/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: user.email,
+        password: user.password,
+      }),
```

Comment:
user state를 함수 밖에서 구조분해할당을 해주면 객체의 key와, value로 작성될 변수가 같으면 하나로 생략할 수 있는 특징을 이용해 다음과 같이 축약할 수 있습니다.
```suggestion
     const {email, password} = user
      body: JSON.stringify({
        email,
        password,
      }),
```
혹은 지금 user객체가 백엔드에서 요구하는 객체와 동일한 형태를 띄고 있기때문에 다음처럼도 작성할 수 있습니다.
```suggestion
      body: JSON.stringify(user),
```